extension/src/popup/components/sendPayment/SendSettings/Settings/index.tsx

```diff
@@ -85,6 +88,24 @@ export const Settings = ({
   // dont show memo for regular sends to Muxed, or for swaps
   const showMemo = !isSwap && !isMuxedAccount(destination);
   const showSlippage = (isPathPayment || isSwap) && !isSoroswap;
+  const isSendSacToContract =
+    isContractId(destination) &&
+    !isContractId(getAssetFromCanonical(asset).issuer);
```

Comment:
The logic here is basically:
If you're sending to a C address, but you're trying to send a classic asset, grab the C address of the SAC